docs/learn/learn-account-advanced.md

```diff
@@ -9,6 +9,44 @@ slug: ../learn-account-advanced
 
 import RPC from "./../../components/RPC-Connection";
 
+## System Accounts
+
+:::info
+
+You can view system accounts on [Subscan](https://polkadot.subscan.io/account_list?role=module).
+
+:::
+
+As the word suggests, system accounts are used by the system. They are used, for example, for the
+treasury, as storage of crowdloan funds, and nomination pools. Those accounts they will never issue
+an [extrinsic](./learn-extrinsics.md) since they are not accounts held by users and thus do not have
+key pair.
+
+The `Account` column in the list of system accounts on
+[Subscan](https://polkadot.subscan.io/account_list?role=module) shows the ID for each system
+account. Some of those ID are Substrate pallet IDs. For example, the treasury pallet id is
+[`py/trsry`](https://github.com/paritytech/polkadot/blob/6282def1bb053858522cf551b86b2d07aad04f29/runtime/polkadot/src/lib.rs#L818).
+
+It is possible to derive account addresses from these pallet IDs in the same way parachain IDs can
+also be represented as addresses. For example, using the
+[Substrate JS Utilities tool](https://www.shawntabrizi.com/substrate-js-utilities/) by Shawn
+Tabrizi, converting `py/trsry -> address` will return the Substrate address for the treasury:
+
+`5EYCAe5ijiYfyeZ2JJCGq56LmPyNRAKzpG4QkoQkkQNB5e6Z`
+
+Given the multi-chain property of the ecosystem, such address can be mirrored to all parachains. At
+the time of writing, on Subscan the treasury address has assets or on-chain data on 29 networks.
+Note there is no inherent reason that the mirrored address will have treasury logic associated with
+it on those networks.
+
+Different types of system accounts also include:
+
+- `py/cfund` for crowdloans (Substrate pallet ID)
+
+- `X` for parathreads where X is the parathread number
```

Comment:
these are wrong

Reply:
in Tabrizi's Tool :

if I type in Module ID `py/cfund` I get `5EYCAe5ijiYdg22N9CWUvWYfa7De8WmxXdP78KZxhy7N91uv` --> `modlpy/cfund:0`

if I type the number `2000` in the para ID field, I get `5Ec4AhPUwPeyTFyuhGuBbD224mY85LKLMSqSSo33JYWCazU4` --> `para:2000`

any advice on how can I address your feedback? Is the wording wrong?

Reply:
@joepetrowski I changed the content based on https://substrate.stackexchange.com/questions/536/how-do-treasury-accounts-compare-to-end-user-accounts-in-frame and kept it simple. Let us know if the content in the PR now is factually correct.